packages/pdf/pyproject.toml

```
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "rpaframework-pdf"
-version = "4.0.0"
+version = "4.1.0"
```

Nit:
```suggestion
version = "4.0.1"
```
since we don't add a new feature but fix/enhance current behaviour/structure/docs only.